pkg/api/node.go

```diff
@@ -46,22 +45,11 @@ func (h *Handler) GetNodes(w http.ResponseWriter, r *http.Request) {
 	nodes := make([]model.Node, len(clusterInventory.Nodes))
 	i := 0
 	for nodeID, n := range clusterInventory.Nodes {
-		// look up all the disks that the current node has applied OSDs on
-		appliedIDs, err := osd.GetAppliedOSDs(nodeID, h.context.EtcdClient)
-		if err != nil {
-			logger.Errorf("failed to get applied OSDs for node %s: %+v", nodeID, err)
-			w.WriteHeader(http.StatusInternalServerError)
-			return
-		}
-
 		storage := uint64(0)
 		for _, d := range n.Disks {
-			for _, uuid := range appliedIDs {
-				if d.UUID == uuid {
-					// current disk is in applied OSD set, add its storage to the running total
-					storage += d.Size
-				}
-			}
+			// Add up the space of all devices.
+			// We should have a separate metric for osd devices, but keep it simple for now.
+			storage += d.Size
```

Comment:
after looking into this further, the device usage will be changing with the next issue i'm working on https://github.com/rook/rook/issues/304, so i'll hold off on this for now